apps/system/test/marionette/browser_video_test.js

```diff
@@ -35,7 +35,7 @@ marionette('Video', function() {
 
   // https://moztrap.mozilla.org/manage/case/6073/
   test('Confirm video playback', function() {
-    var videoUrl = server.url('VID_0001.mp4');
+    var videoUrl = server.url('VID_0001.webm');
```

Comment:
Are we removing the old `VID_0001.mp4`? It looks like not. It's kind confusing to have two completely different videos `VID_0001.webm` and `VID_0001.mp4`. Let's just rename the new one.